leader-election-impl/src/main/java/com/palantir/leader/proxy/AwaitingLeadershipProxy.java

```diff
@@ -103,6 +110,7 @@ public void run() {
 
     private void gainLeadership() {
         try {
+            leaderLog.info("Quiescing");
```

Comment:
This is a bit too clever - not sure the average reader will understand what you mean. Might be best to report something like `"Waiting to become the leader..."`.